charts/aws-mountpoint-s3-csi-driver/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: aws-mountpoint-s3-csi-driver
 description: A Helm chart for installing the Mountpoint for Amazon S3 CSI Driver. This CSI driver allows your Kubernetes applications to access S3 objects through a file system interface.
-version: 1.3.1
+version: 1.4.0
```

Comment:
these changes will come later, you don't need them for this PR

Reply:
Removed